sdk/src/platform/Platform.ts

```diff
@@ -624,6 +627,7 @@ export interface SendOptions {
     method?: string;
     query?: any;
     headers?: any;
+    userAgent?: string;
```

Comment:
Why it should be done on per-request basis and not per-instance?

Reply:
Hi @kirill-konshin For some SDK wrapper, such as RingCentral Call Control JS SDK, JS SDK instance is passed from outside of  Call Control JS SDK. We want to track Call Control  SDK usage with X-USER-AGENT, but we can't change JS SDK instance's x-user-agent directly. If SDK wrapper changed it, every request including request out of Call Control JS SDK will have same x-user-agent.